2018-09-13 10:06:59,092 - DataBase.py[line:23] - INFO: database class is creating...
2018-09-13 10:06:59,306 - DataBase.py[line:51] - INFO: database is connecting...
2018-09-13 10:09:09,974 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_id=000001
2018-09-13 10:09:10,022 - DataBase.py[line:56] - INFO: select password from empinfo where Emp_id=000001
2018-09-13 10:09:29,798 - DataBase.py[line:67] - INFO: INSERT INTO `facepro`.`empinfo` (`Emp_id`, `Emp_name`, `Emp_department`, `password`, `Emp_photo`) VALUES ('000000', 'name1', 'SEU', '000000', 'None')
2018-09-13 10:09:29,835 - DataBase.py[line:80] - ERROR: update database exception, (1062, "Duplicate entry '000000' for key 'PRIMARY'")
2018-09-13 10:10:00,838 - DataBase.py[line:67] - INFO: INSERT INTO `facepro`.`empinfo` (`Emp_id`, `Emp_name`, `Emp_department`, `password`, `Emp_photo`) VALUES ('000001', 'name2', 'SEU', '000001', 'None')
2018-09-13 10:10:00,877 - DataBase.py[line:80] - ERROR: update database exception, (1062, "Duplicate entry '000001' for key 'PRIMARY'")
2018-09-13 10:15:12,292 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_id=000000
2018-09-13 10:15:20,292 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_id=000001
2018-09-13 10:15:24,483 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_id=000001
2018-09-13 10:15:33,683 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_id=000000
2018-09-13 10:15:43,828 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_id=000001
2018-09-13 10:44:01,935 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_id=000001
2018-09-13 10:44:01,984 - DataBase.py[line:56] - INFO: select password from empinfo where Emp_id=000001
